### DISCUSSION TITLE
Acesso à dados com .NET, C#, Dapper e SQL Serve. Falha entre aulas: Dapper:UpDate X Imersão execute Many

### DISCUSSION THREAD

**Discussion by AleLucasG** [balta-io/forum#559] [Tira Dúvidas] [answered]
Boa noite! Estou fazendo o curso:  Acesso à dados com .NET, C#, Dapper e SQL Serve. Acabei de concluir a aula: Dapper - Update.
Como pode ser visto nos prints abaixo, a aula termina ensinando apenas a métodos de UpdateCategory.

![image](https://github.com/balta-io/forum/assets/95228437/29cc7e26-97e5-41a3-a9bd-489328f673ea)

![image](https://github.com/balta-io/forum/assets/95228437/56d41327-0f80-45f3-b703-deba488d05ea)

![image](https://github.com/balta-io/forum/assets/95228437/28124343-d96b-46fe-a99f-99d3daf0cea9)

Quando a próxima aula inicia: Imersão: Execute Many. A mesma começa com métodos que não foram ensinados e um conteudo a frente. Como se tivesse um salto ou seria um erro? 
Métodos que foram pulados na aula: GetCategorie e DeleteCategoria.

Ao final da aula: Execute Many, ele diz que "Amazon AWS" não tem. E quando isso foi deletado????

![image](https://github.com/balta-io/forum/assets/95228437/a6a66cd3-7567-4a2a-9465-ffddc9fb31d6)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @AleLucasG 
Eu conferi aqui e no repositório estão presentes os métodos, acredito que não gravei estas aulas, só deixei os exemplos por serem muito parecidos com o create:

https://github.com/balta-io/2806/blob/main/Modulo%202/Program.cs

Veja se fica alguma dúvida por favor!

**Comment by AleLucasG** [comment]
Olá! muito obrigada! Sem duvidas.
